README.md

```diff
@@ -152,17 +159,16 @@ cat error.log | kubectl-ai "explain the error"
 
 You can use the following special keywords for specific actions:
 
-* `model`: Display the currently selected model.
-* `models`: List all available models.
-* `version`: Display the `kubectl-ai` version.
-* `reset`: Clear the conversational context.
-* `clear`: Clear the terminal screen.
-* `exit` or `quit`: Terminate the interactive shell (Ctrl+C also works).
+- `model`: Display the currently selected model.
```

Comment:
Why do we need this change? The rendered result is the same.

Reply:
This part was automatically converted by the VSCode editor. I'm not sure which Extension completed it, but it automatically formatted the README to make it more organized.